redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -56,17 +58,29 @@ import platform.UIKit.UITableViewDataSourceProtocol
 import platform.UIKit.UITableViewDelegateProtocol
 import platform.UIKit.UITableViewRowAnimationNone
 import platform.UIKit.UITableViewScrollPosition
+import platform.UIKit.UITableViewStyle
 import platform.UIKit.UIView
 import platform.UIKit.indexPathForItem
 import platform.UIKit.item
 import platform.darwin.NSInteger
 import platform.darwin.NSObject
 
-internal open class UIViewLazyList(
-  internal val tableView: UITableView = UITableView(
+internal open class UIViewLazyList() : LazyList<UIView>, ChangeListener {
+  internal val tableView: UITableView = object : UITableView(
     CGRectZero.readValue(),
-  ),
-) : LazyList<UIView>, ChangeListener {
+    UITableViewStyle.UITableViewStylePlain,
+  ) {
+    override fun setContentOffset(contentOffset: CValue<CGPoint>, animated: Boolean) {
+      // If the caller is requesting a contentOffset with y == 0,
+      // and the current contentOffset.y is not 0,
+      // assume that it's a programmatic scroll-to-top call.
+      if (contentOffset.useContents { y } == 0.0 && this.contentOffset.useContents { y } != 0.0) {
+        isDoingProgrammaticScroll = animated
```

Comment:
Based on the comment I would have assumed this would be set to `true`. Or maybe to change the value passed to `super` to be `true`?

Reply:
I did this to be consistent with [this line](https://github.com/cashapp/redwood/blob/7925bec1d6b135be5975a1d54ee36b6a3381a3cd/redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt#L117), but yeah, I'd rather have it always be `true`. Will change.